packages/jsii/test/deprecated-remover.test.ts

```diff
@@ -0,0 +1,161 @@
+import { compileJsiiForTest } from '../lib';
+
+const DEPRECATED = '/** @deprecated stripped */';
```

Comment:
Why is this additional `stripped` needed? Why not just `/** @deprecated */`

Reply:
`/** @deprecated */` is in jsii required to have a deprecation note, or it'll generate an error... So I juts put something here...